src/modules/advanced-custom-attributes/index.js

```diff
@@ -0,0 +1,191 @@
+			} ).join( ' ' ) :
+			''
+	)
+
+	const [ error, setError ] = useState( '' )
+
+	useEffect( () => {
+		const timeout = setTimeout( () => {
+			try {
+				if ( customAttributes.trim() === '' ) {
+					props.onChange( [] )
+					setError( '' )
+					return
+				}
+
+				let tempCustomAttributes = customAttributes
+				const customAttributesMatch = customAttributes.match( /(.*?(?=\=))\="[^\"]*"/g )
+				if ( customAttributes.trim() !== '' && ! customAttributesMatch ) {
+					throw new Error( __( 'Not a valid attribute. Input must be of pattern key1="value1" key2="value2".', i18n ) )
+				}
+
+				// Trim the original input value to check if there are unnecessary characters added.
+				customAttributesMatch.forEach( attribute => {
+					tempCustomAttributes = tempCustomAttributes.replace( attribute.trim(), '' )
+				} )
+
+				if ( tempCustomAttributes.trim() !== '' ) {
+					throw new Error( sprintf( __( "Unexpected syntax: '%s'.", i18n ), tempCustomAttributes.trim() ) )
+				}
+
+				const newCustomAttributes = []
+				customAttributesMatch.forEach( attribute => {
+					const [ key, ..._value ] = attribute.trim().split( '=' )
+					const value = JSON.parse( _value.join( '=' ) )
+
+					/**
+					 * Checks if the attribute key and value can be a valid html attribute.
+					 * Throws an error if not valid.
+					 */
+					document.createElement( 'div' ).setAttribute( key, value )
+
+					/**
+					 * Checks if the attribute key and value can be a valid react prop
+					 * Throws an error if not valid.
+					 */
+					try {
+						const dummyNode = document.createElement( 'div' )
+						render( <div { ...{ [ key ]: value } } />, dummyNode )
+						unmountComponentAtNode( dummyNode )
+
+						if ( INVALID_HTML_ATTRIBUTES.includes( key ) ) {
+							throw new Error()
+						}
+					} catch {
+						throw new Error( sprintf( __( "Attribute key '%s' is not allowed.", i18n ), key ) )
+					}
+
+					newCustomAttributes.push( [ key, value ] )
+				} )
+
+				props.onChange( newCustomAttributes )
+				setError( '' )
+			} catch ( e ) {
+				setError( e.toString().trim() )
+				return false
+			}
+		}, 300 )
+
+		return () => clearTimeout( timeout )
+	}, [ customAttributes ] )
+
+	return (
+		<Fragment>
+			<TextControl
+				data-testid="custom-attributes"
+				label={ __( 'Custom Attributes', i18n ) }
+				value={ customAttributes }
+				onChange={ setCustomAttributes }
+			/>
+			{ !! error && <p className="stk-custom-attributes-control__has-error">{ error }</p> }
+		</Fragment>
+	)
+}
+
+const addInspectorControls = ( output, props ) => {
+	return (
+		<Fragment>
+			{ output }
+			<PanelAdvancedSettings
+				title={ __( 'Custom Attributes', i18n ) }
+				initialOpen={ false }
+			>
+				<p className="components-base-control__help">
+					{ __( 'Add custom HTML attributes to your Stackable blocks.', i18n ) }
+					&nbsp;
+					<a href="https://docs.wpstackable.com" target="_docs">{ __( 'Learn more about Custom Attributes', i18n ) }</a>
+				</p>
```

Comment:
Instead of adding this here, add this as the `help` prop of the `CustomAttributesControl`

Also change the text to:
```js
help={ __( 'You can type in custom HTML attributes for this block below.', i18n ) }
```